meta/client.go

```diff
@@ -0,0 +1,92 @@
+package meta
+
+import (
+	"time"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/influxdb/influxdb/influxql"
+	"github.com/influxdb/influxdb/meta/internal"
+)
+
+type Client struct{}
```

Comment:
Client is the interface from the nodes to the consistent store. Properties need to be added to this type but it's just a placeholder for the mutator methods below.